src/ansiblelint/testing/__init__.py

```diff
@@ -18,7 +18,8 @@
 
 # Emulate command line execution initialization as without it Ansible module
 # would be loaded with incomplete module/role/collection list.
-prepare_environment()
+runtime = Runtime(isolated=True)
+runtime.prepare_environment()
```

Comment:
I don't really like this pattern of initializing a single behaviour with two separate things/invocations which must always be one after another with no code causing side-effects in between.

Reply:
You will have to get used to it because I remember @tadeboro suggesting months ago that doing extra optional stuff inside constructor was a bad idea. The two step process is there by design and not accident, so we have a simple constructor.